src/components/Modal.module.scss

```diff
@@ -0,0 +1,24 @@
+@use '@/partials/variables' as *;
+
+.modalBackdrop {
+  width: 100%;
+  height: 100vh;
+  position: absolute;
+  background-color: rgba($color: #000000, $alpha: 0.5);
+  top: 0;
+  cursor: pointer;
+}
+
+.modalWrapper {
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.modalContainer {
+  background-color: $white;
+  border-radius: 6px;
+  padding: 32px;
+  cursor: default;
+}
```

Comment:
A good place to define widths for different breakpoints since this is a modal container